wetmap/src/components/newModals/diveSiteImage/style.module.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../styles/variables.scss';
+
+.pictureBoxQ {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  margin-bottom: 1vh;
+  border-radius: 1vw;
+  position: relative;
+}
+
+.helper {
```

Comment:
Please try to think about more meaningful class names.

Reply:
Added new style names which more closely represent the containers they contain.